### PR TITLE
Minor changes to code style and documentation

### PR DESCRIPTION
**Pull request by J535D165**
This PR proposes to fix some PEP8 issues, add some docstrings, and clarify the examples/documentation at some points.

### CODE REVIEW

**Comment by qubixes** [comment]
Thanks. I must admit I'm not so much a fan of changing the WSS/RRF to lower case; it will break a number of scripts + I don't see all that much benefit. But if you insist, you can merge it.

**Comment by J535D165** [comment]
We can make this backward compatible if you like.

**Comment by qubixes** [comment]
Sure, let's do that. Apart from that everything looked good to me. Thanks again.

**Comment by J535D165** [comment]
Any idea @qubixes why the 0.7.2 tests suddenly fail? Seems to be unrelated to this PR.

File: README.md [hunk 1]
```diff
@@ -2,36 +2,37 @@
...
-Test this by:
+After installation of the visualization package, `asreview` should automatically
+detect it. Test this by:
 
 ```bash
 asreview --help
 ```
 
-It should list the 'plot' modus.
+It should list the 'plot' extension.
```
Comment (qubixes) [inline README.md:25]: Well, I would say that "plot" is a modus, while asreview-visualization is an extension (with possibly more than a single modus).

File: asreviewcontrib/visualization/plot_discovery.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 class PlotDiscovery(PlotBase):
     def __init__(self, analyses, result_format="percentage"):
+        """Base class for the Discovery plot."""
```
Comment (qubixes) [inline asreviewcontrib/visualization/plot_discovery.py:6]: Base class -> Class ?

File: asreviewcontrib/visualization/plot_inclusions.py [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+
 import numpy as np
 
 from asreviewcontrib.visualization.plot_base import PlotBase
 
 
 class PlotInclusions(PlotBase):
     def __init__(self, analyses, result_format="percentage", thick=None):
-        """
+        """Base class for the Inclusions plot.
```
Comment (qubixes) [inline asreviewcontrib/visualization/plot_inclusions.py:10]: Same as the other

File: asreviewcontrib/visualization/plot_inclusions.py [hunk 4]
```diff
@@ -115,8 +146,8 @@ def add_random(self, text_at=None, col='black', add_text=True):
             xlim = self.ax.get_xlim()
             ylim = self.ax.get_ylim()
             text_at = (
-                np.average(xlim) - 0.07 * (xlim[1]-xlim[0]),
-                np.average(ylim) + 0.07 * (ylim[1]-ylim[0]),
+                np.average(xlim) - 0.07 * (xlim[1] - xlim[0]),
+                np.average(xlim) + 0.07 * (ylim[1] - ylim[0]),
```
Comment (qubixes) [inline asreviewcontrib/visualization/plot_inclusions.py:150]: Why the change in behavior?

File: asreviewcontrib/visualization/plot_limit.py [hunk 5]
```diff
@@ -4,8 +4,11 @@
 
 
 class PlotLimit(PlotBase):
-    def __init__(self, analyses, prob_allow_miss=[0.1, 0.5, 2.0],
+    def __init__(self,
+                 analyses,
+                 prob_allow_miss=[0.1, 0.5, 2.0],
                  result_format="percentage"):
+        """Base class for the Limit plot."""
```
Comment (qubixes) [inline asreviewcontrib/visualization/plot_limit.py:11]: Same

File: asreviewcontrib/visualization/plot_progression.py [hunk 6]
```diff
@@ -3,14 +3,19 @@
...
 class PlotProgression(PlotBase):
-    def __init__(self, analyses, result_format="percentage", thick=None,
-                 sigma=25, window=40):
+    def __init__(self,
+                 analyses,
+                 result_format="percentage",
+                 thick=None,
+                 sigma=25,
+                 window=40):
+        """Base class for the Regression plot."""
```
Comment (qubixes) [inline asreviewcontrib/visualization/plot_progression.py:18]: Same

File: asreviewcontrib/visualization/plot_inclusions.py (hunk 4 above)
Comment (J535D165) [reply asreviewcontrib/visualization/plot_inclusions.py]: Resolved.

**Review by qubixes — APPROVED** [review]
